### ISSUE TITLE
Content announced incorrectly

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#58346]
### Issue Description:

The screen reader is announcing the content inaccurately, which can lead to confusion for users relying on assistive technologies. This mismatch between the screen reader’s output and the visual presentation creates an inconsistent experience, making it difficult for users to understand the content or navigate the interface effectively. The numbers available of items for each color are not announced.

<img width="1138" alt="Image" src="https://github.com/user-attachments/assets/cb6a41df-ddd4-4586-b8aa-927d0756825b" />

<img width="309" alt="Image" src="https://github.com/user-attachments/assets/2e464837-9889-4a8b-9c0a-c450b8cc924b" />

### Component or Page: 

Filters (v.9.9)

### Name affected element:

Numbers inside each filter option

### Relevant code snippets:

```
Code example: 
<span class=""wc-block-product-filter-chips__label"">
                                                        <span class=""wc-block-product-filter-chips__text"">
                                                                Purple                                                        </span>
                                                                                                                        <span class=""wc-block-product-filter-chips__count"">
                                                                        (1)
                                                                </span>
                                                                                                        </span>
```

### Recommended fix:

Make sure the information presented visually is conveyed the same way through the screen reader. Review and align both the visual design and accessibility elements to ensure consistency.

Remove the aria-label from the button so all content within is announced or add the count to the aria-label.
